pkg/util/mount/mount.go

```diff
@@ -143,6 +142,23 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	return refs, nil
 }
 
+// GetMountRefsByDev finds all references to the device provided
+// by devPath; returns a list of paths.
+func GetMountRefsByDev(mounter Interface, devPath string) ([]string, error) {
+	mps, err := mounter.List()
+	if err != nil {
+		return nil, err
+	}
+	// Find all references to the device.
+	var refs []string
+	for i := range mps {
+		if mps[i].Path == devPath || mps[i].Device == devPath {
```

Comment:
Here is the problem, I get two pieces of info, 1 globalPath which is location where local volume is mounted and dir, which a mounting point for pod where local disk will be mounted. 
In mounter structure globalPath corresponds to Path element.  SO that is why I compare Path with devPath, maybe devPath name is a bit confusing. the second part of if is needed for unit test to work properly as in fake mounter Device element matches to globalPath.